feature-libs/order/components/order-details/order-details.module.ts

```diff
@@ -42,7 +41,6 @@ const moduleComponents = [
     I18nModule,
     FeaturesConfigModule,
     PromotionsModule,
-    OrderOverviewModule,
```

Comment:
we still need this module, but import it from `storefrontlib`.

Reply:
Absolutely, good catch!